### ISSUE TITLE
i18n support.

### ISSUE THREAD

**Issue by yosuke-furukawa** [google/node-sec-roadmap#27]
there are so good documentations.
I would like to translate this docs to Japanesea for Japanese Noders.
We think gitlocalize is very helpful for this type docs. https://gitlocalize.com/
If you are fine for translations, we will change this folder organizations.

**Comment by yosuke-furukawa** [comment]
CC: @sotayamashita @chikathreesix

**Comment by mikesamuel** [comment]
Sounds good.  What can I do to help?

**Comment by sotayamashita** [comment]
@yosuke-furukawa I appreciate your suggestion. :100:

@mikesamuel Hi! I am an engineer for Gitlocalize. We should think how to translate docs first regardless of whether we use GitLocalize or not. Any idea?

**Comment by sotayamashita** [comment]
@mikesamuel 

> If Gitlocalize helps extract human language text and track which commits have changes that need to be translated, that would probably help a lot.

Yes, we do those things. In addition, we return translated file via a pull request to your repository and each translation is associated with GitHub user so they can see what they have been contributed to GitHub. :sparkles:

> I can probably handle redirecting requests for / based on Accept-Language headers.

I got it. I will try to configure. Please wait. :bow:

cc:/ @yosuke-furukawa

**Comment by sotayamashita** [comment]
@mikesamuel I would appreciate if you could tell me the reason why do you write this document under **Google** org.

**Comment by mikesamuel** [comment]
@sotayamashita My employer required it.

Can we set up a fork that just tracks google/node-sec-roadmap and which uses your GH plugin?

**Comment by sotayamashita** [comment]
@mikesamuel 

> My employer required it.

I got it.

> Can we set up a fork that just tracks google/node-sec-roadmap and which uses your GH plugin?

Before forking the repo, I would appreciate if you could follow these conditions below,  I can give you our free OSS plan.  You do not need to fork if it could be accepted. 

- The OSS projects should be non-profitable
- Up to 5 repositories
- Put a note that GitLocalize is used for a localization and its project link on README.md
- If you have a website for a project, put a GitLocalize logo on the site. 

By the way,  [google/WebFundamentals](https://github.com/google/WebFundamentals)  have been using GitLocalize already.

**Comment by mikesamuel** [comment]
> * The OSS projects should be non-profitable

Agreed

> * Up to 5 repositories

Thanks

> * Put a note that GitLocalize is used for a localization and its project link on README.md

Agreed

> * If you have a website for a project, put a GitLocalize logo on the site.

Could it go in the sidebar above "Published with Gitbook"?

<img width="294" alt="screen shot 2018-01-24 at 12 12 57 am" src="https://user-images.githubusercontent.com/368886/35315632-6950988c-009b-11e8-84d2-e6291e5fec8a.png">

**Comment by sotayamashita** [comment]
@mikesamuel 

> Could it go in the sidebar above "Published with Gitbook"?

Sound good. What about to add "Translated by GitLocalize"?

**Comment by mikesamuel** [comment]
@sotayamashita, Sounds good to me.

**Comment by sotayamashita** [comment]
@mikesamuel Please let me know when you integrate this repository with GitLocalize.

**Comment by mikesamuel** [comment]
In offline conversation

Sota from GitLocalize said
> As I mentioned before, google/WebF are using our tool so I ask them to integrate the app
> on GitLocalize. I let you know when it has done.

so I'll try to move this forward once github.com/google/WebFundamentals clicks the necessary checkboxes.

**Comment by mikesamuel** [comment]
@sotayamashita Should I follow up with WebFundamentals?

**Comment by sotayamashita** [comment]
@mikesamuel I will check the status
